src/vs/workbench/contrib/notebook/browser/extensionPoint.ts

```diff
@@ -42,16 +42,20 @@ export interface INotebookRendererContribution {
 	readonly [NotebookRendererContribution.optionalDependencies]: readonly string[];
 }
 
-enum NotebookMarkdownRendererContribution {
+enum NotebookMarkupRendererContribution {
 	id = 'id',
 	displayName = 'displayName',
 	entrypoint = 'entrypoint',
+	dependsOn = 'dependsOn',
+	mimeTypes = 'mimeTypes',
 }
 
-export interface INotebookMarkdownRendererContribution {
-	readonly [NotebookMarkdownRendererContribution.id]?: string;
-	readonly [NotebookMarkdownRendererContribution.displayName]: string;
-	readonly [NotebookMarkdownRendererContribution.entrypoint]: string;
+export interface INotebookMarkupRendererContribution {
+	readonly [NotebookMarkupRendererContribution.id]?: string;
+	readonly [NotebookMarkupRendererContribution.displayName]: string;
+	readonly [NotebookMarkupRendererContribution.entrypoint]: string;
+	readonly [NotebookMarkupRendererContribution.dependsOn]: string | undefined;
```

Comment:
In generic output renderers we call this `dependencies`, we should consider doing so here for consistency (even if its length will be <= 1)